db/migrate/20160307161402_add_scheduled_reports.rb

```diff
@@ -0,0 +1,13 @@
+class AddScheduledReports < ActiveRecord::Migration
+  def change
+    create_table :scheduled_reports do |t|
+      t.string :name, null: false
+      t.string :frequency, null: false
```

Comment:
wouldn't it be possible to do that with AR enum? the enum vals would be long but it seems like working with strings that act like enums would be slightly hairier than working with enums.